engine/classes/Elgg/I18n/Translator.php

```diff
@@ -587,6 +601,36 @@ public function getAvailableLanguages() {
 		return _elgg_services()->hooks->trigger('languages', 'translations', [], $languages);
 	}
 	
+	/**
+	 * Returns an array of allowed languages as configured by the site admin
+	 *
+	 * @return []
```

Comment:
`@return string[]`

`[]` is not valid, see https://docs.phpdoc.org/latest/references/phpdoc/types.html